client/src/App.jsx

```diff
@@ -8,15 +8,15 @@ import {
   Routes,
   Route,
 } from 'react-router-dom';
-import LandingPage from './Components/LandingPage';
-import CameraCapture from './Components/Camera/CameraCapture';
-import Leaderboard from './Components/Leaderboard';
-import Profile from './Components/Profile/Profile';
-import SuccessfulSubmission from './Components/SuccessfulSubmission/SuccessfulSubmission';
-import Register from './Components/Register';
-import Login from './Components/Login';
-import PageNotFound from './Components/PageNotFound';
-import Navbar from './Components/Navbar';
+import LandingPage from './components/LandingPage';
```

Comment:
i see the lowercase change here but the folder seems to still be uppercase, should the folders be then lowercase?

Reply:
I did not realize casing on Github does not change is the letters are all the same. I manually changed it for the components and mockData folders.